internal/fs/inode/file_test.go

```diff
@@ -682,3 +682,33 @@ func (t *FileTest) SetMtime_SourceObjectMetaGenerationChanged() {
 	ExpectEq(newObj.Generation, o.Generation)
 	ExpectEq(newObj.MetaGeneration, o.MetaGeneration)
 }
+
+func (t *FileTest) ContentEncodingGzip() {
+	// Set up an explicit content-encoding on the backing object and re-create the inode.
+	const contentEncoding = "gzip"
```

Comment:
Sorry I mean If it's const initialise it on top with upper case start latter, or here you can initilize it with 
```var contentEncoding string```.

same with others

Reply:
Done.